protocol/src/main/java/org/fidoalliance/fdo/protocol/To2ClientService.java

```diff
@@ -335,11 +335,11 @@ protected void doSetupDevice(Composite request, Composite reply) {
     if (getStorage().getMaxOwnerServiceInfoMtuSz() != null) {
       try {
         ownerMtu = Integer.parseInt(getStorage().getMaxOwnerServiceInfoMtuSz());
-        if (ownerMtu <= 0) {
+        if (ownerMtu < 0) {
```

Comment:
Let's simplify the checks here and keep the minimum MTU size at 1300 bytes.

Suggestion:
```
if (mtu_rcv < DEFAULT_SERVICE_INFO_MTU_SIZE) // 1300
     mtu = DEFAULT_SERVICE_INFO_MTU_SIZE;
else if (mtu_rcv > MAX_SERVICE_INFO_MTU_SIZE) // 8192
    mtu = MAX_SERVICE_INFO_MTU_SIZE;
else
    mtu = mtu_rcv;
```
The check for exception case is correct.